identity/terraform/locals.tf

```diff
@@ -30,7 +30,6 @@ locals {
       AUTH0_CLIENT_ID     = data.aws_ssm_parameter.auth0_client_id[env_name].value
       AUTH0_CALLBACK_URL  = data.aws_ssm_parameter.auth0_callback_url[env_name].value
       API_BASE_URL        = data.aws_ssm_parameter.api_base_url[env_name].value
-      CONTEXT_PATH        = data.aws_ssm_parameter.context_path[env_name].value
```

Comment:
I don't have full confidence that I know what the consequences of removing this are. Could someone with terraform chops check?

Reply:
If you apply this pre-new build is deployed, it will potentially break if we spin up an image with this being required.

Might be worth pulling this out until we've deployed this new version (no harm in it being there) and then removing it